omero/users/history.txt

```diff
@@ -1,155 +1,72 @@
-updating code before the full public release of 5.3.0. Use at your own risk**.
-
-Changes include:
-
-- bumped Bio-Formats version to 5.3.4. This includes several bug fixes.
-  See `Bio-Formats 5.3.4 release history <https://www.openmicroscopy.org/site/support/bio-formats5.3/about/whats-new.html>`_
-- work on the Web API
-- filter rating in Web
-- adjustment to support the upcoming Java 1.9
-- utils script classes deprecated
-- deprecate shares
-- deprecate search bridges
-- open viewer in new tabs
-- rendering engine bug fixes
-- bugs fixes in OMERO.insight and OMERO.web
-- introduce hide-password option in CLI
-- handle correctly shape color 
-- enable annotation search on wells
-- improve reading of tables data
-
-5.3.0-m8 (February 2017)
-------------------------
-
-Developer preview release - **only intended as a developer preview for
-updating code before the full public release of 5.3.0. Use at your own risk**.
-
-Changes include:
-
-- bumped Bio-Formats version to 5.3.3. This includes several bug fixes.
-  See `Bio-Formats 5.3.3 release history <https://www.openmicroscopy.org/site/support/bio-formats5.3/about/whats-new.html>`_
-- work on the Web API
-- removal pdf doc generation
-- improvement to how colors are assigned to channels
-- disabled jquery cache
-- support for projection along T
-- fixed numerous tests
-- bug fixes in OMERO.web
-
-
-5.3.0-m7 (January 2017)
------------------------
-
-Developer preview release - **only intended as a developer preview for
-updating code before the full public release of 5.3.0. Use at your own risk**.
-
-Changes include:
-
-- bumped Bio-Formats version to 5.3.2. This includes several bug fixes.
-  See `Bio-Formats release history <https://www.openmicroscopy.org/site/support/bio-formats5.3/about/whats-new.html>`_
-- work on the Web API
-- handled shapes color in clients according to model specification
-- improved open_with option in web
-- added histogram support in clients and server
-- added integration of all official scripts
-- improved SPW grid view in web
-- bug fixes
-
-5.3.0-m6 (November 2016)
-------------------------
-
-Developer preview release - **only intended as a developer preview for
-updating code before the full public release of 5.3.0. Use at your own risk**.
-
-Changes include:
-
-- bumped Bio-Formats version to 5.3.0-m2. This includes JPEG-XR support for CZI.
-  See the `Bio-Formats version history <https://www.openmicroscopy.org/site/support/bio-formats5.3/about/whats-new.html>`_ for more information.
-- fixed numpy-pillow version incompatibility
-- made python testing package public so it can be used by external clients
-
-5.3.0-m5 (November 2016)
-------------------------
+5.3.0 (March 2017)
+------------------
 
-Developer preview release - **only intended as a developer preview for
-updating code before the full public release of 5.3.0. Use at your own risk**.
+A full, production-ready release of OMERO 5.3.0; featuring a major re-working
```

Comment:
I'll fix that when I'm pushing further changes thanks Mark, as I'm sure they'll be more!